### ISSUE TITLE
Setup guide

### ISSUE THREAD

**Issue by anubra266** [ProfZijlstra/video_site#58]
Would be lovely, if the readme has a guide on how to setup/run the project locally

**Comment by ProfZijlstra** [comment]
Yes, I agree -- not sure yet when I'll do this, but it's definitely something I should make!

**Comment by ProfZijlstra** [comment]
Easy initial fix is to put a blank settings.php file in the root directory so that people can fill in the blanks to get most of the way there

**Comment by okumujustine** [comment]
> Easy initial fix is to put a blank settings.php file in the root directory so that people can fill in the blanks to get most of the way there

How do I run the application?
